src/components/ErrorMessage.tsx

```diff
@@ -1,10 +1,12 @@
 /** @jsx jsx */
 import { jsx, css } from "@emotion/react";
 import colors from "../colors";
+import {selectable} from "../commonStyles";
```

Comment:
The braces here are inconsistent with the ones two lines above, but that's a style thing that doesn't really matter I don't think. It does seem to be consistent with a lot of the rest of the codebase (or at least, what I can see in this diff).

Reply:
Yeah that sounds like a fine change. acb7df25